### PR TITLE
Stop running okd unit tests in installer

### PR DESCRIPTION
**Pull request by patrickdillon**
These are broken because we already removed the okd images. Just need to stop running them.

### CODE REVIEW

File: ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml [hunk 1]
```diff
@@ -8881,7 +8881,7 @@ presubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
-    trigger: (?m)^/test( | .* )okd-unit,?($|\s.*)
+    trigger: (?m)^/test( | .* )(okd-unit|remaining-required),?($|\s.*)
```
Comment (r4f4) [inline ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml:8884]: Is this some c&p typo?

**Comment by r4f4** [comment]
If we are not building any images, we should just remove the okd variant altogether.

**Review by r4f4 — COMMENTED** [review]
/approve

**Comment by patrickdillon** [comment]
pushed new commit with updated mirror mappings based on the failing test

**Review by r4f4 — COMMENTED** [review]
/lgtm

**Pull request closed without merging**